cdap-data-fabric/src/main/java/co/cask/cdap/data/stream/CoordinatorStreamProperties.java

```diff
@@ -45,6 +47,7 @@ public String toString() {
       .add("notificationThresholdMB", getNotificationThresholdMB())
       .add("generation", getGeneration())
       .add("description", getDescription())
+      .add("ownerPrincipal", getOwnerPrincipal())
```

Comment:
We decided that if there is no owner present we will return null to the user  rather than showing the inferred owner.